### PR TITLE
[Tests-only]Fix core test run structure in drone

### LINKED ISSUE #526: make testing core test not depend on configurations in user_ldap
currently new contexts and new suites of core tests need to be added to the test config in user_ldap, it would be better if that would not be needed

### PR DESCRIPTION
**Pull request by dpakach**
fixes #526 
Run All of the core tests by using the behat.yml config from core.

With this we dont have to change suites and contexts in user_ldap if they change in core.

### CODE REVIEW

**Comment by dpakach** [comment]
089a06b94e53b4ab11a8fe2bc97a78dec8343cb3 just removes `'cron': 'nightly'` from the pipelines so that we can see all the tests passing. Once they pass, that commit will be dropped before merging.

File: .drone.starlark [hunk 1]
```diff
@@ -520,15 +436,15 @@ config = {
...
 		},
 		'core-webui-acceptance': {
-			'suites': [
-				'webUICore1',
-				'webUICore2',
-				'webUICore3',
-				'webUICore4',
-			],
+			'suites': {
+				'webUIall': 'core-webUI',
```
Comment (kiranparajuli589) [inline .drone.starlark]:
```suggestion
				'webUIAll': 'core-webUI',
```

File: tests/acceptance/config/behat.yml [hunk 2]
```diff
@@ -2,458 +2,24 @@ default:
...
-    apiUserLDAP:
-      paths:
-        - '%paths.base%/../features/apiUserLDAP'
-      context: *common_ldap_suite_context
-      contexts:
-        - UserLdapGeneralContext:
-        - UserLdapUsersContext:
-        - FeatureContext: *common_feature_context_params
         - LoggingContext:
         - OccContext:
```
Comment (kiranparajuli589) [inline tests/acceptance/config/behat.yml]: :+1:

File: tests/acceptance/config/behat.yml [hunk 3]
```diff
@@ -2,458 +2,24 @@ default:
   autoload:
     '': '%paths.base%/../features/bootstrap'
 
-  suites:
```
Comment (kiranparajuli589) [inline tests/acceptance/config/behat.yml]: why removing `suites`?

File: tests/acceptance/config/behat.yml [hunk 4]
```diff
@@ -2,458 +2,24 @@ default:
...
-        - TrashbinContext:
-
-    apiUserLDAP:
-      paths:
-        - '%paths.base%/../features/apiUserLDAP'
-      context: *common_ldap_suite_context
-      contexts:
-        - UserLdapGeneralContext:
-        - UserLdapUsersContext:
-        - FeatureContext: *common_feature_context_params
```
Comment (kiranparajuli589) [inline tests/acceptance/config/behat.yml]: :hocho: haripp @HariBhandari07

File: .drone.starlark (hunk 1 above)
Comment (HariBhandari07) [reply .drone.starlark]: @kiranparajuli589 Dipak is using the camelcase naming convention, so it's fine.

**Comment by dpakach** [comment]
Drone passed when running all the tests including nightly builds.
https://drone.owncloud.com/owncloud/user_ldap/2364

**Pull request merged**